charts/consul/templates/crd-gatewaypolicies.yaml

```diff
@@ -199,42 +199,93 @@ spec:
             - targetRef
             type: object
           status:
+            description: GatewayPolicyStatus defines the observed state of the gateway
             properties:
               conditions:
-                description: Conditions indicate the latest available observations
-                  of a resource's current state.
+                default:
+                - lastTransitionTime: "1970-01-01T00:00:00Z"
+                  message: Waiting for controller
+                  reason: Pending
+                  status: Unknown
+                  type: Accepted
+                - lastTransitionTime: "1970-01-01T00:00:00Z"
+                  message: Waiting for controller
+                  reason: Pending
+                  status: Unknown
+                  type: Programmed
```

Comment:
So, this default doesn't match the valid conditions that the controller sets, I think you'll want to change it to `ResolvedRefs`

Reply:
good  catch! I had copied/pasted this and forgot to change it :facepalm: